packages/common/index.ts

```diff
@@ -208,23 +208,47 @@ export enum OpenQuestionStatus {
   Drafting = "Drafting",
   Queued = "Queued",
   Helping = "Helping",
-  CantFind = "CantFind",
-  TADeleted = "TADeleted",
+  PriorityQueued = "PriorityQueued",
+}
+
+/**
+ * Limbo statuses are awaiting some confirmation from the student
+ */
+export enum LimboQuestionStatus {
+  CantFind = "CantFind", // represents when a student can't be found by a TA
+  ReQueueing = "ReQueueing", // represents when a TA wants to get back to a student later and give them the option to be put into the priority queue
+  TADeleted = "TADeleted", // When a TA deletes a question for a multitude of reasons
 }
 
 export enum ClosedQuestionStatus {
   Resolved = "Resolved",
-  Deferred = "Deferred",
   ConfirmedDeleted = "ConfirmedDeleted",
+  StudentCancelled = "StudentCancelled",
   Stale = "Stale",
 }
```

Comment:
Should we get rid of Open Limbo and Closed statuses and just have a single QuestionStatus enum now that we have the StatusIn arrays?

Reply:
@ofloody thoughts? I still kind of like the way it is right now -- since it helps delineate to a reader what each question status stands for.